src/main/menus/help.js

```diff
@@ -2,10 +2,7 @@ import { shell } from 'electron'
 import * as actions from '../actions/help'
 import { checkUpdates } from '../actions/marktext'
 
-const notOsx = process.platform !== 'darwin'
-const updateMenuVisibility = process.platform === 'win32' || !!process.env.APPIMAGE
-
-export default {
+let helpMenu = {
```

Comment:
Use `const` is preferred.

Reply:
Updated. I've used style from [menus/view](/marktext/marktext/blob/master/src/main/menus/view.js#L3)